base/subarray.jl

```diff
@@ -143,43 +143,43 @@ reindex(V, idxs::Tuple{DroppedScalar, Any, Vararg{Any}}, subidxs::Tuple{Any, Any
 
 # In general, we simply re-index the parent indices by the provided ones
 getindex(V::SubArray) = (@_propagate_inbounds_meta; getindex(V, 1))
-function getindex(V::SubArray, I::Int...)
+function getindex(V::SubArray, I::Real...)
```

Comment:
We did; the deprecation is still in effect, though, so this way SubArray behaves as it did before my refactor and still issues warnings.